Dockerfile.base

```diff
@@ -1,64 +1,38 @@
 # syntax=docker/dockerfile:1
 
 # BASE docker image for music assistant container
-# Based on Debian Trixie (testing) because we need a newer version of ffmpeg (and snapcast)
-# TODO: Switch back to regular python stable debian image + manually build ffmpeg and snapcast ?
-FROM debian:trixie-slim
-
-ARG TARGETPLATFORM
+#
+# It should contain ONLY packages we want to deploy in the final image, not
+# packages needed to build. Build dependencies should be installed in the
+# "builder" image in the main Dockerfile.
 
+FROM python:3.12-alpine3.21
 
 RUN set -x \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends \
+    && apk add --no-cache \
         ca-certificates \
-        curl \
-        git \
-        wget \
```

Comment:
curl, wget, git are needed in the final image

Reply:
Oh, what for? (just curious)

Reply:
> Oh, what for? (just curious)

ffmpeg uses curl (or wget, I cant remember which) for streaming from http, at least that is what I remember.
git is used to install dependencies from a git source, something we still allow (for now)

Reply:
hmm, maybe it does no longer use curl or wget nowadays, we'll have to test